src/gentropy/method/colocalisation.py

```diff
@@ -179,6 +179,11 @@ def colocalise(
                     f.sum(f.col("clpp")).alias("clpp"),
                 )
                 .withColumn("colocalisationMethod", f.lit(cls.METHOD_NAME))
+                .join(overlapping_signals.calculate_beta_ratio(),
+                 ["leftStudyLocusId",
+                  "rightStudyLocusId",
+                   "chromosome"],
+                "left")
```

Comment:
I know, these are positional arguments, but I like to be more explicit in joins by providing 

```python
.join(
    overlapping_signals.calculate_beta_ratio(),
    on=["leftStudyLocusId", "rightStudyLocusId","chromosome"],
    how="left"
)
```